lib/orgmode.js

```diff
@@ -272,6 +273,26 @@ class Orgmode extends OutlinesArrayList {
     }, {});
   }
 
+  ///**
+  // * @property {Object} overview - the overview of this document
+  // */
+  //get blocks() {
+  //  return this._ast.options.reduce((map, item) => {
+  //    map[item.name.toLowerCase()] = item.value;
+  //    return map;
+  //  }, {});
+  //}
+
+  /**
+   * @method findBlockByName(name)
+   * @param {Number} name
+   * @return {Block} return the blocks
+   */
+  findBlockByName(name) {
+    return this._ast.blocks.filter(
+      (block) => block.name === name
+    );
+  }
```

Comment:
@yorkie The class `OutlinesArrayList` don't have field `_ast`, how can I move  `findBlockByName` to class `OutlinesArrayList`?